utils.go

```diff
@@ -122,22 +126,26 @@ func (s *Storage) formatError(op string, err error, path ...string) error {
 	}
 }
 
-func (s *Storage) formatFileObject(v *upyun.FileInfo) (o *types.Object, err error) {
-	o = &types.Object{
-		ID:         v.Name,
-		Name:       s.getRelPath(v.Name),
-		Type:       types.ObjectTypeFile,
-		Size:       v.Size,
-		UpdatedAt:  v.Time,
-		ObjectMeta: info.NewObjectMeta(),
-	}
+func (s *Storage) formatFileObject(v *upyun.FileInfo) (o *typ.Object, err error) {
+	o = s.newObject(false)
```

Comment:
Please make sure the `done`'s value is correct, or user will fall in endless `stat` call as object will never be marked as `true`